cloudbuild.yaml

```diff
@@ -23,29 +23,87 @@ steps:
   # build gke-exec-auth-plugin binary
   - name: 'gcr.io/cloud-builders/bazel'
     args:
-      - --output_base=/workspace/plugin-binaries
+      - --output_user_root=/workspace/bazel-root
```

Comment:
we should still add these flags to the steps above so we can keep the cache across them